packages/kit/src/core/postbuild/prerender.js

```diff
@@ -205,23 +205,30 @@ async function prerender({ out, manifest_path, metadata, verbose, env }) {
 		/** @type {Map<string, import('types').PrerenderDependency>} */
 		const dependencies = new Map();
 
-		const response = await server.respond(new Request(config.prerender.origin + encoded), {
-			getClientAddress() {
-				throw new Error('Cannot read clientAddress during prerendering');
-			},
-			prerendering: {
-				dependencies
-			},
-			read: (file) => {
-				// stuff we just wrote
-				const filepath = saved.get(file);
-				if (filepath) return readFileSync(filepath);
-
-				// stuff in `static`
-				return readFileSync(join(config.files.assets, file));
-			},
-			emulator
-		});
+		const response = await server.respond(
+			new Request(config.prerender.origin + encoded, {
+				headers: {
+					accept: 'text/html,*/*'
```

Comment:
Should we add a comment here explaining why we do this?

Reply:
I'll add a comment (although, I'm not quite sure how to best phrase it)